src/vcModals.cpp

```diff
@@ -242,7 +242,19 @@ void vcModals_SetTileImage(void *pProgramStatePtr)
   vcState *pProgramState = (vcState*)pProgramStatePtr;
 
   char buf[256];
-  udSprintf(buf, "%s/0/0/0.%s", pProgramState->settings.maptiles.tileServerAddress, pProgramState->settings.maptiles.tileServerExtension);
+  switch (pProgramState->settings.maptiles.serverFlag)
+  {
+  case vcMTSF_Unknown:
+	  break;
+  case vcMTSF_Slippy:
+	  udSprintf(buf, "%s/0/0/0.%s", pProgramState->settings.maptiles.tileServerAddress, pProgramState->settings.maptiles.tileServerExtension);
+	  break;
+  case vcMTSF_Google:
+	  udSprintf(buf, "%s?lyrs=s&hl=zh-CN&gl=CN&x=0&y=0&z=0", pProgramState->settings.maptiles.tileServerAddress);
+	  break;
+  default:
+	  break;
```

Comment:
When iterating an enum; we encourage using all values and not using `default` in order to cause compilation issues when adding additional enum values in future